.github/workflows/test-unit-gh_runner.yml

```diff
@@ -52,9 +53,43 @@ jobs:
         timeout-minutes: 10 # generous, expected time is approx 1 min.
         shell: "bash"
         run: |
+          set +e     # Ignore intermediate error(s) in this step
```

Comment:
Is this going to mean that it will no longer stop after 1st unittest failure, and could run through all (generating a core file for each potentially) ?

That may be okay, but what is the intent of the set +e here?

Reply:
If "Bash"specified, then "bash -e {0}" executed internally. You can claim it need to fail fast, then I will remove the 'set +e'.